src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -370,13 +370,15 @@ private bool CanConnect(object parameter)
         // Handler to invoke node Auto Complete
         private void AutoComplete(object parameter)
         {
-            DynamoViewModel dynamoViewModel = _node.DynamoViewModel;
-            var svm = dynamoViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel;
-
-            svm.PortViewModel = parameter as PortViewModel;
-            //svm.PopulateAutoCompleteCandidates();
-
-            dynamoViewModel.CurrentSpaceViewModel.ShowNodeAutoCompleteSearchCommand.Execute(ShowHideFlags.Show);
+            // Since NodeAutoCompleteCommand is not triggered from xml event trigger directly,
+            // CanAutoComplete will not be triggered automatically.
+            if (CanAutoComplete(parameter))
```

Comment:
I don't see this pattern used for other commands where the `Can` method is being called explicitly so why make an exception in this case?

Reply:
@aparajit-pratap Please refer to the comment, all the other view commands here are directly triggered by WPF xml event trigger but this one is not. This command is triggered by code by calling execute(). Calling execute() does not call `CanAutoComplete` at all. That is why my original change worked but after moving the logic to `CanAutoComplete`, it regressed. We can consider removing this command and simply implement a function as an alternative.